src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,905 @@
-::The Gutter - Intro <1156,67>
-Anna was right. That piece of shit was trying to kill me one way or another. I guess I’d known it all along.
+<!-- If using debug menu shortcuts, prompts player to enter their name -->
+
+::The Gutter - Name prompt
+{ (set: $playerName to (prompt: "Enter name:", ""))
+(if: (text: $playerName)'s length > 0)[Right, my name is $playerName. [[Moving on|The Gutter - Outside approach]].{
+  (if: $playerName's last is "s")[
+    (set: $playerNamePlural to ($playerName + "es"))
+  ]
+  (else-if: ($playerName's last is "h") and ($playerName's 2ndlast is "s"))[
+    (set: $playerNamePlural to ($playerName + "es"))
+  ]
+  (else-if: ($playerName's last is "h") and ($playerName's 2ndlast is "c"))[
+    (set: $playerNamePlural to ($playerName + "es"))
+  ]
+  (else:)[
+    (set: $playerNamePlural to ($playerName + "s"))
+  ]
+}]
+}(else:)[No, [[that's not right|The Gutter - Name prompt]].]
+
+(set: $dissolve to (transition: 'dissolve') + (transition-time: 3s))
+(set: $hatchet to true)(set: $multitool to true)
+
+::The Gutter - Outside approach
+My body and mind were screaming as one.
+
+(click: "one")[$dissolve[I kept my head down and my sweatshirt hood up. I used all of my energy to move forward through the tarp-covered alleyways of Gatherville.
+
+My gut lurched as I realized I’d need to ask around for more information on Anna. The last thing I wanted to do was talk to anyone else.]]
+
+(click: "talk to anyone else")[$dissolve[I pulled my screaming brain into the harsh light of reality, searching for answers through a veil of emotion. It was still pretty early in the night. Around this time, anyone who knew anything was:
+
+>>[[Getting fucked up at the Gutter|The Gutter - Outside intro]]
+>>[[Looking to score at the Roach Market|RM-first-entrance]]
+>>[[Catching an early show at the Church|Church - intro]]]]
+
+(set: $gutterOutside to "intro")(set: $spokeToToe to false)(set: $roachEntrance to "first")
+
+::The Gutter - Outside intro
+I stood outside a weathered, flat-roofed wooden building with a flickering neon sign.
+<!--This calls one of the three neon sign designs randomly every second. -->
+\
+(click: "neon sign")[(live: 1s)[(either: $guttersign1, $guttersign2, $guttersign3)]
+<!--This makes the following passage dissolve in under the sign after the player has been in the passage for 5 seconds. -->
```

Comment:
This timing stuff was really tough to gauge while testing, because it's hard to force myself to read at a player speed. Can definitely look at adjusting this after play tests or if everyone else thinks it's too long. My idea was definitely to let the sign hang by itself for a beat or so.

Reply:
Oh, it's actually 10 seconds in the code right now. I must've adjusted it up without adjusting the note. My bad.

Reply:
Even five seconds feels a hair long to me. Let's play around with 3-5 and see how that feels? I wonder if we could also accelerate the rate of change on the sign to make it feel more like an actual flickering neon sign — the fade in/out is a little _too_ smooth, you know?

Reply:
Tried out these suggestions and:
* Three seconds for the message after the sign works really well!
* Changed the live: macro on the sign to variate every .4 seconds instead of 1 second and it looks better.